src/utils/suggestion.ts

```diff
@@ -0,0 +1,54 @@
+import store from '@/store'
+import { VueRenderer } from '@tiptap/vue-3'
+import tippy from 'tippy.js'
+import { Instance, Props } from 'tippy.js'
+import MentionList from '../components/MentionList.vue'
+
+export default {
+  items: ({ query }) => {
+    const users = store.getters["asana/users"]
+    return users.filter(u => u.name.toLowerCase().startsWith(query.toLowerCase())).slice(0, 5)
+  },
+
+  render: () => {
+    let component: VueRenderer;
+    let popup: Instance<Props>[];
+
+    return {
```

Comment:
Did this get copied from somewhere on the internet?

Reply:
Yes it's from [here](https://tiptap.dev/api/nodes/mention).

Reply:
Please add that link in this file at the top